CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- **Textarea** character countdown (optional)
```

Comment:
This could be a `5.4.2`

Reply:
@kevinch releasy now updates the `changelog`, adding automatically the correct version and date